materials/module-1/presentation/part-1/home-internet.md

```diff
@@ -0,0 +1,21 @@
+# Home internet
+
+```
+                                                +---------+
+                                                | Dropbox |
+                                                | data    |
+                                                | centre  |
+                                                +---------+
+                                                     ^
+                                                     |
+                                                     | Internet
+                                                     |
++---------+                                     +----+-----+
+| Your    |          Last-mile network          | Internet |
+| home    +------------------------------------>| service  |
+| devices |                                     | provider |
++---------+  https://dropbox.com/some-resource  +----------+
```

Comment:
Consider not using ASCII images to have them render consistently across screen

Reply:
Done